src/CallbackStream.php

```diff
@@ -40,12 +40,20 @@ public function __construct(callable $callback)
      */
     public function __toString()
     {
+        ob_start();
```

Comment:
Hmm no that would defeat the purpose. I mainly just wanted to show that it's possible to get the benefit of direct output, without breaking the API contract.